README.md

```diff
@@ -1 +1,9 @@
-# docs
\ No newline at end of file
+# Docs
+
+Docs for accounts.
+
+## Getting Started
+
+Install gitbook `npm i -g gitbook-cli`.
+
+Run the server `gitbook serve`
```

Comment:
Instead of requiring global install do `yarn init` and `yarn add --dev gitbook-cli`.
Also override the `start` script with the `gitbook serve` command